src/contentTransformations.ts

```diff
@@ -1 +1,4 @@
-export const transform = (html: string): string => html ? html.replace(/•/g, '<span class="bullet">•</span>') : '';
+export const transform = (html: string): string => html
+    .replace(/•/g, '<span class="bullet">•</span>')
+    .replace(new RegExp(/\<h2\>\* \* \*\<\/h2\>/, 'g'), '<hr class="section__rule"/>')
```

Comment:
Why does CAPI provide us with an `<h2>` with some asterisks inside it here 🤔? Is it intended to be a heading semantically or is it always interpreted as an `<hr>`? Is this defined by CAPI or Composer?

@regiskuckaertz @JustinPinner  @jonathonherbert are you able to shed any light on this?

Reply:
Do you have the article ID? Probably entered as is by the journalist.

Reply:
`news/2019/dec/16/dreaming-of-a-green-christmas-heres-how-to-make-it-come-true`

I thought this was a convention used as I've seen it on a few articles and read dotcom mentioning it too 🤔

Reply:
Spotted again today: https://www.theguardian.com/society/2019/dec/17/decade-of-perpetual-crisis-2010s-disrupted-everything-but-resolved-nothing